test/e2e/test-cmd.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Retry 10 times with 10s interval to wait for the apiserver to come back.
+function wait-for-apiserver()
+{
+  for count in {0..9}; do
+    kubectl version && rc=$? || rc=$?
+    if [ ${rc} -eq 0 ]; then
+      return 0
+    else
+      sleep 10
+    fi
+  done
+  return 1
+}
+
+# Sanity check.
+# Note that the log inidicates that the kubectl in the test driver is v1.10.7
+kubectl version
+
+# TODO: in the real test, create objects here.
+
+# Change the apiserver manifest to change the storage version.
+gcloud compute --project "${PROJECT}" ssh --zone "${KUBE_GCE_ZONE}" "${CLUSTER_NAME}-master" --command \
+"sudo sed -i \"s/--v=/--storage-versions=apps\/v1beta2 --v=/\" /etc/kubernetes/manifests/kube-apiserver.manifest"
```

Comment:
don't hard-code v1beta2?

Reply:
I will lock the apiserver version in the job config. That should make this hardcoding reliable.